apps/service_providers/messaging_service.py

```diff
@@ -76,20 +78,30 @@ def send_whatsapp_voice_message(self, synthetic_voice: SynthesizedAudio, from_nu
                 "ContentType": "audio/mpeg",
             },
         )
-        public_url = self.s3_client.generate_presigned_url(
+        return self.s3_client.generate_presigned_url(
             "get_object",
             Params={
                 "Bucket": settings.WHATSAPP_S3_AUDIO_BUCKET,
                 "Key": file_path,
             },
             ExpiresIn=360,
         )
-        self.client.messages.create(from_=f"whatsapp:{from_number}", to=f"whatsapp:{to_number}", media_url=[public_url])
+
+    def send_text_message(self, message: str, from_: str, to: str, platform: ChannelPlatform):
+        prefix = self.TWILIO_CHANNEL_PREFIXES[platform]
+        self.client.messages.create(from_=f"{prefix}:{from_}", body=message, to=f"{prefix}:{to}")
+
+    def send_voice_message(self, synthetic_voice: SynthesizedAudio, from_: str, to: str, platform: ChannelPlatform):
+        prefix = self.TWILIO_CHANNEL_PREFIXES[platform]
+        public_url = self._upload_audio_file(synthetic_voice)
+        self.client.messages.create(from_=f"{prefix}:{from_}", to=f"{prefix}:{to}", media_url=[public_url])
 
     def get_message_audio(self, message: TwilioMessage) -> BytesIO:
         auth = (self.account_sid, self.auth_token)
-        ogg_audio = BytesIO(requests.get(message.media_url, auth=auth).content)
-        return audio.convert_audio(ogg_audio, target_format="wav", source_format="ogg")
+        response = requests.get(message.media_url, auth=auth)
+        # Example header: {'Content-Type': 'audio/ogg'}
+        content_type = response.headers["Content-Type"].split("/")[1]
```

Comment:
what other content types might we expect?

Reply:
So when we get a voice message from whatsapp through twilio, its `audio/ogg`, but when we get it from Messenger through twilio, its `video/mp4`.